.changeset/silly-hats-end.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/abi-typegen": patch
```

Comment:
Should it be a minor that it became a breaking change?
```suggestion
"@fuel-ts/abi-coder": minor
"@fuel-ts/abi-typegen": minor
```

Reply:
➕